cms/tests/functions.php

```diff
@@ -0,0 +1,88 @@
+<?php
+use PHPUnit\Framework\TestCase;
+
+require '../functions.php';
+
+
+class StackTest extends TestCase
+{
+    //Success
+    public function testReturnProjectTitles()
+    {
+        $expected = "Pilot Shop\nCMS Project\nNew Project\n\n\n\n\n\n\n";
+        $input =[0=>["id"=>"1","projectTitle"=>"Pilot Shop","projectTitleLink"=>"https://dev.maydenacademy.co.uk/students/2018/osama/PilotShopSite/","projectImage"=>"https://dev.maydenacademy.co.uk/students/2018/osama/Portfolio/assets/PilotShop.png"],1=>["id"=>"2","projectTitle"=>"CMS Project","projectTitleLink"=>"https://dev.maydenacademy.co.uk/students/2018/osama/PilotShopSite/", "projectImage"=>"https://dev.maydenacademy.co.uk/students/2018/osama/Portfolio/assets/PilotShop.png"],2=>["id"=>"3","projectTitle"=>"New Project","projectTitleLink"=> "https://dev.maydenacademy.co.uk/students/2018/osama/PilotShopSite/","projectImage"=>"https://dev.maydenacademy.co.uk/students/2018/osama/Portfolio/assets/PilotShop.png"],3=>["id"=>"4","projectTitle"=>NULL,"projectTitleLink"=>NULL,"projectImage"=>NULL],4=>["id"=>"5","projectTitle"=> NULL,"projectTitleLink"=>NULL,"projectImage"=>NULL],5=>["id"=>"6","projectTitle"=>NULL,"projectTitleLink"=>NULL,"projectImage"=>NULL],6=>["id"=>"7","projectTitle"=>NULL,"projectTitleLink"=>NULL,"projectImage"=>NULL],7=>["id"=>"8","projectTitle"=>NULL,"projectTitleLink"=>NULL,"projectImage"=>NULL],8=>["id"=>"9","projectTitle"=>NULL,"projectTitleLink"=>NULL,"projectImage"=>NULL]];
```

Comment:
can make the input array shorter as it only has to be dummy data

Reply:
same for all tests

Reply:
these tests will all become one when you change your functions into one in your code